openstack/dcs/v2/tags/requests.go

```diff
@@ -0,0 +1,79 @@
+package tags
+
+import "github.com/chnsz/golangsdk"
+
+var RequestOpts = golangsdk.RequestOpts{
+	MoreHeaders: map[string]string{"Content-Type": "application/json", "X-Language": "en-us"},
+}
+
+//ResourceTag is in key-value format
+type ResourceTag struct {
+	Key string `json:"key" required:"ture"`
+	// Value is not required, but it must be included in the parameter.
+	Value string `json:"value"`
+}
+
+type createTagsOpts struct {
```

Comment:
suggest to rename this struct as it was used by Create and **Delete** methods.

Reply:
yes, good idea